### PR TITLE
fix(common-labels): Specify SemVer version

### PR DESCRIPTION
**Pull request by jai**
Fixes https://github.com/kubernetes/website/issues/32855

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/common-labels.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ on every resource object.
 | ----------------------------------- | --------------------- | -------- | ---- |
 | `app.kubernetes.io/name`            | The name of the application | `mysql` | string |
 | `app.kubernetes.io/instance`        | A unique name identifying the instance of an application | `mysql-abcxzy` | string |
-| `app.kubernetes.io/version`         | The current version of the application (e.g., a semantic version, revision hash, etc.) | `5.7.21` | string |
+| `app.kubernetes.io/version`         | The current version of the application (e.g., a SemVer 1.0, revision hash, etc.) | `5.7.21` | string |
```
[the remark this answers is not in the document]
Comment (jai) [reply content/en/docs/concepts/overview/working-with-objects/common-labels.md:40]: ah yeah I was looking for a way to link, didn't realize regular markdown was supported 😂 thanks for that!

[the remark this answers is not in the document]
Comment (jai) [reply content/en/docs/concepts/overview/working-with-objects/common-labels.md]: fixed/updated

**Comment by jai** [comment]
@tengqm @sftim seems we need an /lgtm label as well (I think)

**Comment by tengqm** [comment]
/lgtm
again

**Pull request merged**